lib/server.js

```diff
@@ -0,0 +1,92 @@
+AntiSearchSource.allow = function(collectionName, allowRules) {
+  check(allowRules, {
+    maxLimit: Match.Optional(Number),
+    securityCheck: Match.Optional(Function),
+    allowedFields: Match.Optional([String]),
+    queryTransform: Match.Optional(Function)
+  });
+  this._allowRules[collectionName] = allowRules;
+  return this._allowRules[collectionName];
+};
+
+AntiSearchSource._SearchConfig = Match.Where(function(config) {
```

Comment:
@IuriiBod this is redundant. May be defined as local cost variable or used directly inside method.